README.md

```diff
@@ -5,3 +5,158 @@
 - Web and Network  
 - Software Development
 - Web Development
+
+### BASIC COMPUTER TERMS
+
+#### Computer
+A computer is a device that executes a program or programs. 
```

Comment:
A computer is a machine or device that performs processes, calculations and operations based on instructions provided by a software or hardware program. They are designed to execute applications and provide a variety of solutions by combining integrated hardware and software components. Modern computers include laptop, ATM and also many smart devices (smartphones, smart TV, smart watches, game console and smart appliances etc)